components/file_system/src/lib.rs

```diff
@@ -25,26 +29,28 @@ use std::sync::{Arc, Mutex};
 use openssl::error::ErrorStack;
 use openssl::hash::{self, Hasher, MessageDigest};
 
-#[derive(Debug)]
+#[derive(Clone, Copy, Debug)]
 pub enum IOOp {
     Read,
     Write,
 }
 
-#[derive(Clone, Copy, Debug)]
+#[derive(Clone, Copy, Debug, PartialEq)]
 pub enum IOType {
-    Other,
-    Read,
-    Write,
-    Coprocessor,
-    Flush,
-    Compaction,
-    Replication,
-    LoadBalance,
-    Import,
-    Export,
+    Other = 0,
+    Read = 1,
```

Comment:
is it really necessary?

Reply:
I think this is good. Making the enum -> int mapping explicit.